src/admin/dumb_components/custom_email_form.js

```diff
@@ -0,0 +1,33 @@
+const React = require('react')
+const { map, objOf, zipWith, merge, compose } = require('ramda')
+
+export default class CustomEmail extends React.Component {
```

Comment:
I think let's try and keep `dump_components` as function components. We could put an `onSubmit` on the form and then we can drop references to `this` by looking on the event object